ipywidgets/widgets/widget_box.py

```diff
@@ -75,6 +75,22 @@ class PlaceProxy(Proxy):
     selector = Unicode(sync=True)
 
 
+def VBox(*pargs, **kwargs):
+    """Displays multiple widgets vertically using the flexible box model."""
+    box = Box(*pargs, **kwargs)
+    box.layout.display = 'flex'
+    box.layout.flex_direction = 'column'
+    return box
+
+
+def HBox(*pargs, **kwargs):
```

Comment:
It is not using `FlexBox` anymore.

These helper functions are much more used than `Box` or `FlexBox` around me (which are almost never used directly). I don't think that we should deprecate it.